utils/testman/tests/changeset_test.go

```diff
@@ -0,0 +1,178 @@
+package tests
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/acorn-io/manager/tests/e2e"
+	"github.com/acorn-io/manager/tests/helper"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func randToken(length int) (string, error) {
+	randBytes := make([]byte, length)
+
+	_, err := rand.Read(randBytes)
+	if err != nil {
+		return "", err
+	}
+
+	return hex.EncodeToString(randBytes), nil
+}
+
+func trimToDir(path, targetDir string) (string, error) {
+	// Split the path into its components
+	components := strings.Split(path, string(filepath.Separator))
+
+	// Find the index of the target directory
+	targetIndex := -1
+	for i, component := range components {
+		if component == targetDir {
+			targetIndex = i
+			break
+		}
+	}
+
+	// If the target directory is not found, return an error
+	if targetIndex == -1 {
+		return "", fmt.Errorf("target directory '%s' not found in path", targetDir)
+	}
+
+	// Construct the new path from the beginning to the target directory
+	newPath := filepath.Join(components[:targetIndex+1]...)
+	return string(filepath.Separator) + newPath, nil
+}
+
+func loadChangesetMap(path string) (map[string]string, error) {
+	fullChangesets, err := os.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+
+	var obj any
+	err = json.Unmarshal(fullChangesets, &obj)
+	if err != nil {
+		return nil, err
+	}
+
+	jsonMap, ok := obj.(map[string]interface{})
```

Comment:
Using `any` and `interface{}`.

Reply:
Changed all `interface{}` types to any. I didn't think to change those after generating that code.